app/views/event_frequencies/index.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.array! @event_frequencies, partial: "event_frequencies/event_frequency", as: :event_frequency
```

Comment:
investigate this does

Reply:
I believe this was a bit of rails magic that was created when I created the event frequencies scaffold